### ISSUE TITLE
Log viewer advanced usage

### ISSUE THREAD

**Issue by MPeti1** [MuntashirAkon/AppManager#495]
- [x] I know what my device, OS and App Manager versions are
- [x] I know how to take logs
- [x] I know how to reproduce the issue which may not be specific to my device

**Describe the bug**
Filtering in the log viewer is not working as expected, because it seemingly does not affect the log output. If I include a new log tag in one of the filter menus (3 dots -> filters or 3 dots -> settings -> log viewer -> filter out tags), I will still see logs containing this text, but others too.

**To Reproduce**
Steps to reproduce the behaviour:
1. Open the log viewe (main app list menu -> 3 dots -> log viewer)
2. wait until you see the logs scrolling
3. add a new filter in either of the above described places
4. watch for new log entries, and notice that there's no difference

**Expected behavior**
The list of shown log entries should change according to the filter settings.

**Screenshots**

**Device info**
 - Device: [e.g. Redmi 3S]
 - OS Version: [e.g. Android 6]
 - App Manager Version: [e.g. v2.7.0-1222-DEBUG (386)] 
 - Mode: root

**Comment by MuntashirAkon** [comment]
Log viewer is implemented exactly the way it is implemented in Catlog/Matlog.

**Filter out tags:** As the name suggests, it would remove the specified tags from the list.

**Filters:** It stores a list of filters which can be applied by clicking on them. They aren't applied when you add them to the list.

**Comment by MPeti1** [comment]
oh, I see how the *Filters* work, didn't know about this feature, but *Filter out tags* does not seem to work, and I think it also didn't work in Matlog.

I use this filter expression:
```
ResourceType|memtrack|android.os.Debug|BufferItemConsumer|DPM.*|MDM.*|ChimeraUtils|BatteryExternalStats.*|chatty.*|DisplayPowerController|WidgetHelper|WearableService|DigitalWidget.*|^ANDR-PERF-.*|WtComponentManager|TetherStatsReporting|(Phone)?StatusBar
```
This is the default, and some additional tags separated by a `|` sign.

An example is that TetherStatsReporting appears a lot if GlassWire is running, and it appears even though it's included in the filters

**Comment by MuntashirAkon** [comment]
Yeah. My tests have suggested the same which is why I haven't closed the issue.

**Comment by MuntashirAkon** [comment]
It seems it doesn't actually filters out the logs, it simply convert them to verbose (`V`).

**Comment by MPeti1** [comment]
It's an interesting approach. Would it be possible to make it to work as an actual filter?

**Comment by MuntashirAkon** [comment]
Fixed in a9204ba3 and 9620ed08.
